toolbackup/src/main/java/io/dockstore/toolbackup/client/cli/FormattedTimeGenerator.java

```diff
@@ -10,24 +10,31 @@
 /**
  * Created by kcao on 11/01/17.
  */
-public class FormattedTimeGenerator {
+public final class FormattedTimeGenerator {
+
     private static final int MIN_IN_H = 60;
 
+    private FormattedTimeGenerator() {
+        // hide constructor
+    }
+
     public static String getFormattedTimeNow(LocalDateTime now) {
         DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
         return now.format(formatter);
     }
-    public static Date strToDate(String dateStr){
+
+    public static Date strToDate(String dateStr) {
         try {
             return new SimpleDateFormat("dd-MM-yyyy HH:mm:ss").parse(dateStr);
         } catch (ParseException e) {
             throw new RuntimeException("Could not parse: " + dateStr + " into a date");
         }
     }
+
     public static String elapsedTime(LocalDateTime start, LocalDateTime end) {
         long totalMinutes = ChronoUnit.MINUTES.between(start, end);
-        int minutes = (int)totalMinutes % MIN_IN_H;
-        int hours = ((int)totalMinutes - minutes)/MIN_IN_H;
+        int minutes = (int) totalMinutes % MIN_IN_H;
+        int hours = ((int) totalMinutes - minutes) / MIN_IN_H;
         return hours + " hours and " + minutes + " minutes";
```

Comment:
Super-duper-uber-minor, but we can streamline this code and make it work even if the difference in times is huge:
```
long minutes = totalMinutes % MIN_IN_H;
long hours = totalMinutes / MIN_IN_H;
return hours + " hours and " + minutes + " minutes";
```